scarlet/update.py

```diff
@@ -214,13 +216,106 @@ def translation(component, direction=1, kernel=interpolation.lanczos, padding=3)
     return component
 
 
-def symmetric(component, strength=1, use_prox=True, kernel=interpolation.lanczos, padding=3):
+def psf_weighted_centroid(component):
+    # Extract the arrays from the component
```

Comment:
The descriptor could be a little more explicit. Especially given that it returns a `component` and not arrays.

Reply:
Agreed. This was just copy pasted from Nate's code and I didn't pay too close attention to the (lack of a) docstring.